### ISSUE TITLE
Episode lists not updating and a cache issue

### ISSUE THREAD

**Issue by dtrigleth** [moneymaker365/plugin.video.ustvvod#44]
I have been experiencing an issue for a couple weeks now with new episodes not populating in the episode list.  Whenever there is a new episode of any show that I have a favorite shortcut set for, the list will not update with the newest episode until I restart my system.  I've tried starting from the main USTV screen and navigating to the show, and it still is not updated.

Also, every time I access a network (doesn't matter ABC, CBS, whatever), my system caches the shows.  If I cancel it, the list returns as empty, so I have to let it cache (takes upwards of 5 minutes with each network).  If I leave that network and come right back into it, it "re-caches" but only takes about 30 seconds, but if I access a different network and then come back to the first one, it goes back to the 3-5 minute cache.  Am I doing something wrong?  This is why I have favorite shortcuts for all the shows I watch, so I don't have to waste so much time navigating through the networks.

If you want a log, let me know.  I'm hoping these are known issues and there is either no fix for them or it's a system setting I can change.

**Comment by slices81** [comment]
When you say shortcut is this a xbmc favorite or  a USTV vod favorite. There is an issue with the forner where xbmc caches the list and doesn't call the addon.  Although the fact that you navigate to the show, implies that this is a different issue.  I presume the new episodes are actually available on the website without a cable login?

The caching behavior is correct but  should remain at 30 seconds once it's set.  Full debug log might help, also try setting TV DB lookup to off in settings.

**Comment by dtrigleth** [comment]
Thank you for responding so quickly.

I do use XBMC shortcuts primarily, but I have also (as you already read) tried navigating from the ground up through the USTV add-on, still with no updated episodes.  The episodes are available without any login, and I have no problem with them showing as available once my system restarts.

I currently have the TVDB lookup to "quiet" because I had it to "off" previously but realized I wasn't getting new shows populating.  I'll play around with it set to "off" and see how if the cache time improves.

**Comment by slices81** [comment]
There is no episode caching in the addon, try deleting your favorites and navigate from the addon.

No idea what is happening with the show caching, upload a log.

**Comment by z-e-r-o** [comment]
Please upload log file.
Also give some information about how you connect to the services.

**Comment by dtrigleth** [comment]
I guess the problem is solved then.  I have to either navigate through the add-on or restart my Pi every evening in order to have access to the updated episodes.  Navigating from the ground up will show new episodes, but if I've already accessed the show via an XBMC favorite, then that older list keeps showing no matter what.  I switched my TVDB lookup off, and it's definitely loading into networks faster.  Thanks everyone.
